### PR TITLE
[#9373] Standardize footer design

### LINKED ISSUE #9373: Standardize footer design
Currently V7's footer is still the placeholder created during the initial page skeleton design:
screen shot 2019-01-26 at 1 12 45 am

However, V6 does not fare that much better since it is wildly varying from page to page:
- Static front pages
  screen shot 2019-01-26 at 1 08 42 am
- Data-serving pages
  screen shot 2019-01-26 at 1 09 13 am
- Help pages
  screen shot 2019-01-26 at 1 09 25 am
- Error pages
  screen shot 2019-01-26 at 1 09 41 am

Need to standardize the footer design. Let's use the design used in V6's data-serving pages, i.e.
- Application version
- `[for {institute name}]` (only when applicable)
- Send feedback

"Hosted on Google App Engine" does not value-add even if shown, so it can be dropped.

### PR DESCRIPTION
**Pull request by monmanuela**
Fixes #9373 

Screenshots from #9373:
> Currently V7's footer is still the placeholder created during the initial page skeleton design:
> screen shot 2019-01-26 at 1 12 45 am
> 
> However, V6 does not fare that much better since it is wildly varying from page to page:
> 
> * Static front pages
>   screen shot 2019-01-26 at 1 08 42 am
> * Data-serving pages
>   screen shot 2019-01-26 at 1 09 13 am
> * Help pages
>   screen shot 2019-01-26 at 1 09 25 am
> * Error pages
>   screen shot 2019-01-26 at 1 09 41 am

After my changes in v7, the footer looks like this:
image

**Outline of Solution**

The TEAMMATES version number is obtained from the environment config file.
The institute name is obtained in `GetAuthInfoAction`.

### CODE REVIEW

File: src/main/java/teammates/ui/webapi/action/GetAuthInfoAction.java [hunk 1]
```diff
@@ -40,7 +41,9 @@ public ActionResult execute() {
                     gateKeeper.getLoginUrl(frontendUrl + Const.WebPageURIs.ADMIN_HOME_PAGE)
             );
         } else {
-            output = new AuthInfo(userInfo, authType == AuthType.MASQUERADE,
+            String googleId = userInfo.getId();
+            AccountAttributes accountInfo = logic.getAccount(googleId);
+            output = new AuthInfo(userInfo, accountInfo.getInstitute(), authType == AuthType.MASQUERADE,
```
Comment (wkurniawan07) [inline src/main/java/teammates/ui/webapi/action/GetAuthInfoAction.java]: If `accountInfo` is null, you're going to hit NPE here. Need to add a guard.

File: src/web/app/page.component.html [hunk 2]
```diff
@@ -67,9 +67,14 @@ <h1 *ngIf="pageTitle && isValidUser">{{ pageTitle }}</h1>
   <footer>
     <div class="container footer text-nowrap">
       <div class="row">
-        <div class="col-xs-12 col-sm-4 offset-sm-8 col-md-3 offset-md-9">
-          Hosted on <a class="footer" href="https://cloud.google.com/appengine/" target="_blank" rel="noopener noreferrer">Google App Engine</a>
+        <div class="col-sm-4 col-md-4 text-left">
+          <i class="fas fa-bolt"></i><a routerLink="/web/front/home"> TEAMMATES</a> v{{ version }}
```
Comment (wkurniawan07) [inline src/web/app/page.component.html]: Minor issue, but let's upcase the `v` before version to make it similar with V6.

Comment (monmanuela) [reply src/web/app/page.component.html]: Fixed!

File: src/main/java/teammates/ui/webapi/action/GetAuthInfoAction.java (hunk 1 above)
Comment (monmanuela) [reply src/main/java/teammates/ui/webapi/action/GetAuthInfoAction.java]: Fixed!

**Pull request merged**